spyder/plugins/variableexplorer/widgets/dataframeeditor.py

```diff
@@ -193,6 +194,17 @@ class DataFrameModel(QAbstractTableModel, SpyderFontsMixin):
 
     For more information please see:
     https://github.com/wavexx/gtabview/blob/master/gtabview/models.py
+
+    Attributes
+    ----------
+    bgcolor_enabled : bool
+        If True, vary backgrond color depending on cell value
+    colum_avg_enabled : bool
+        If True, select background color by comparing cell value against
+        column maximum and minimum. Otherwise, use maximum and minimum of
+        the entire dataframe.
+    _format_spec : str
```

Comment:
```suggestion
    format_spec : str
```

Seems a typo?

Reply:
No, the attribute is indeed called `self._format_spec` with an initial underscore; I think I am to blame for this.